### PR TITLE
check tag from attributes

### PR DESCRIPTION
**Pull request by jeanawhou**
@Quiescent @cameronsutter 

These are the steps to reproduce the bug for this fix
1. Add a tag to a character
2. Open the filter menu
3. The tag will not show up on the list

Demo for the fix

https://user-images.githubusercontent.com/19387007/221245096-dd79a450-ca8a-4aad-82f8-73ebc604461e.mp4

### CODE REVIEW

File: lib/pltr/v2/selectors/__tests__/categories.test.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      it('should have 1 filter item', () => {
+        const currentFilterItems = categoriesFilterItemsSelector(stateAfterAddingNote)
+        expect(currentFilterItems.length).toBe(1)
+      })
+      it('should have the uncategorized filter item', () => {
+        const currentFilterItems = categoriesFilterItemsSelector(stateAfterAddingNote)
+        expect(currentFilterItems).toEqual([uncategorized])
+      })
+
+      describe('change category', () => {
```
Comment (Quiescent) [inline lib/pltr/v2/selectors/__tests__/categories.test.js:65]: Is this the correct behaviour?  (We won't be able to filter to uncategorised characters.)

Comment (Quiescent) [reply lib/pltr/v2/selectors/__tests__/categories.test.js:65]: NVM.  I was wrong.

File: lib/pltr/v2/selectors/tagsThirdOrder.js [hunk 2]
```diff
@@ -64,9 +66,21 @@ export const tagsFilterItemsSelector = createSelector(
         return filteredItems
       }
       case 'characters': {
-        const filteredItems = tags.filter((tag) =>
-          characters.find((char) => char.tags?.includes(tag.id))
-        )
+        const filteredItems = tags.filter((tag) => {
```
Comment (Quiescent) [inline lib/pltr/v2/selectors/tagsThirdOrder.js:69]: Can we have a test for this, too, please? :)

Comment (jeanawhou) [reply lib/pltr/v2/selectors/tagsThirdOrder.js:69]: Added :)

Comment (Quiescent) [reply lib/pltr/v2/selectors/tagsThirdOrder.js:69]: Thanks!

**Review by Quiescent — APPROVED** [review]
Good work! :)

**Pull request merged**